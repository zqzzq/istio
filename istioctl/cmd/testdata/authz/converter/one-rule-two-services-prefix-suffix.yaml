apiVersion: "rbac.istio.io/v1alpha1"
kind: ServiceRole
metadata:
  name: service-viewer
  namespace: default
spec:
  rules:
    - services: ["svc-*", "*-xyz"]
      methods: ["GET"]
