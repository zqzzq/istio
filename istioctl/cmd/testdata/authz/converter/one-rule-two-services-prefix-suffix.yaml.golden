===PLEASE REVIEW THE GENERATED POLICY AND REMOVE THIS LINE BEFORE APPLYING IT===
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: global-deny-all
  namespace: my-root-namespace
spec:
  {}
---
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: service-svc-wildcard-rule0-target0
  namespace: default
spec:
  rules:
  - from:
    - source:
        namespaces:
        - istio-system
    to:
    - operation:
        methods:
        - GET
  selector:
    matchLabels:
      app: svc-a-xyz
      version: a
---
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: service-svc-wildcard-rule0-target1
  namespace: default
spec:
  rules:
  - from:
    - source:
        namespaces:
        - istio-system
    to:
    - operation:
        methods:
        - GET
  selector:
    matchLabels:
      app: svc-b-xyz
      version: b
---
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: service-svc-wildcard-rule0-target2
  namespace: default
spec:
  rules:
  - from:
    - source:
        namespaces:
        - istio-system
    to:
    - operation:
        methods:
        - GET
  selector:
    matchLabels:
      app: svc-c-xyz
      version: c
---
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: service-wildcard-xyz-rule0-target0
  namespace: default
spec:
  rules:
  - from:
    - source:
        namespaces:
        - istio-system
    to:
    - operation:
        methods:
        - GET
  selector:
    matchLabels:
      app: svc-a-xyz
      version: a
---
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: service-wildcard-xyz-rule0-target1
  namespace: default
spec:
  rules:
  - from:
    - source:
        namespaces:
        - istio-system
    to:
    - operation:
        methods:
        - GET
  selector:
    matchLabels:
      app: svc-b-xyz
      version: b
---
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: service-wildcard-xyz-rule0-target2
  namespace: default
spec:
  rules:
  - from:
    - source:
        namespaces:
        - istio-system
    to:
    - operation:
        methods:
        - GET
  selector:
    matchLabels:
      app: svc-c-xyz
      version: c
---
